packages/@dcl/sdk-commands/src/commands/quests/index.ts

```diff
@@ -145,14 +144,16 @@ async function executeCreateSubcommand(
         body: JSON.stringify(quest)
       })
       if (res.status === 201) {
-        const questsId = (await res.json()) as { id: string }
-        printSuccess(logger, `> Your Quest: ${quest!.name} was created successfully - ID: ${questsId.id}`, '')
+        const { id: questId } = (await res.json()) as { id: string }
+        components.analytics.track('Quest Created Success', { questId, questName: quest!.name })
+        printSuccess(logger, `> Your Quest: ${quest!.name} was created successfully - ID: ${questId}`, '')
       } else {
-        const error = (await res.json()) as { code: number; message: string }
+        const { code, message } = (await res.json()) as { code: number; message: string }
+        components.analytics.track('Quest Created Failure', { code, message })
```

Comment:
are we sure that we want to send this message ? isn't enough with the code ?
I dont know what this message could be but if we are already sending a code we can then map that failure with the message